store.go

```diff
@@ -815,35 +822,88 @@ func (s *store) GraphDriver() (drivers.Driver, error) {
 // used by the Store.  Accessing this store directly will bypass locking and
 // synchronization, so it is not a part of the exported Store interface.
 func (s *store) LayerStore() (LayerStore, error) {
```

Comment:
Maybe we can mention that `LayerStore` is deprecated and `RWLayerStore` should be used? Or something similar to make clear that it's just an alias.

Reply:
Sure, added a deprecation notice.